use-cases/GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,238 @@
+   $ sudo lspci -nn -D | grep Graphics
+     0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Broadwell-U Integrated Graphics [8086:1616] (rev 09)
+   ```
+
+   Run the above command to find out the BDF for the GPU device on host.<br/>
+   From the above output, PCI addres "0000:00:02.0" is assigned to the hardware GPU device.<br/>
+   This BDF will be used later to unbind the GPU device from host.<br/>
+   "8086 1616" is the device ID of the hardware GPU device. It will be used later to
+   rebind the GPU device to `vfio-pci` driver.
+
+2. Find the IOMMU group for the GPU device:
+
+   ```
+   $ BDF="0000:00:02.0"
+   $ readlink /sys/bus/pci/devices/$BDF/iommu_group
+   ../../../../kernel/iommu_groups/1
+   ```
+
+   The above output shows that the GPU belongs to IOMMU group 1.
+
+3. Unbind the GPU:
+
+   ```
+   $ echo $BDF | sudo tee /sys/bus/pci/devices/$BDF/driver/unbind
+   ```
+
+4. Bind GPU to `vfio-pci` device driver:
+
+   ```
+   $ sudo modprobe vfio-pci
+   $ echo 8086 1616 | sudo tee /sys/bus/pci/drivers/vfio-pci/new_id
+   $ echo $BDF | sudo tee --append /sys/bus/pci/drivers/vfio-pci/bind
+   ```
+
+   After running the above commands, the GPU will be now bound to `vfio-pci` driver.<br/>
+   A new directory with the IOMMU group number will be created under `/dev/vfio`:
+
+   ```
+   $ ls  /dev/vfio
+     1 vfio
+   ```
+
+5. Start a Kata container with GPU device:
+
+   ```
+   $ sudo docker run -it --runtime=kata-runtime --rm  --device /dev/vfio/1  -v /dev:/dev debian /bin/bash
+   ```
+
+   Running `lspci` within the container should show the GPU device among the PCI devices.
+
+   ```
+   $ lspci -nn -D
+     0000:00:00.0 Class [0600]: Device [8086:1237] (rev 02)
+     0000:00:01.0 Class [0601]: Device [8086:7000]
+     0000:00:01.1 Class [0101]: Device [8086:7010]
+     0000:00:01.3 Class [0680]: Device [8086:7113] (rev 03)
+     0000:00:02.0 Class [0604]: Device [1b36:0001]
+     0000:00:03.0 Class [0780]: Device [1af4:1003]
+     0000:00:04.0 Class [0100]: Device [1af4:1004]
+     0000:00:05.0 Class [0002]: Device [1af4:1009]
+     0000:00:06.0 Class [0200]: Device [1af4:1000]
+     0000:00:0f.0 Class [0300]: Device [8086:1616] (rev 09)
+   ```
+
+   Additionally, you should be able to access the device node for the graphics device:
+
+   ```
+   $ ls /dev/dri
+     card0  renderD128
+   ```
+
+## GVT-g with Kata Containers
+
+Follow  the steps below to pass an Intel Graphics device in GVT-d mode to a Kata Container:
+
+1. Find the Bus-Device-Function(BDF) for GPU device:
+
+   ```
+   $ sudo lspci -nn -D | grep Graphics
+     0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Broadwell-U Integrated Graphics [8086:1616] (rev 09)
+   ```
+
+   Run the above command to find out the BDF for the GPU device on host.
+   From the above output, PCI addres "0000:00:02.0" is assigned to the GPU device.
+   This BDF will be used later to unbind the GPU device from host.
+
+2. Choose the MDEV type for VGPU:
```

Comment:
Can you define what "MDEV" and "VGPU" are please?

Reply:
done.